### PR TITLE
Added support for 2017 Taxonomy and fixed issue where incorrect members were being added to resource file.

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Added 2017 Taxonomy
- Sorted resource file
- Fixed issue where improper members were being attributed to incorrect axes.

### Pull request process

- [ ] Full description of changes provided above.
- [ ] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/resources/DQC_US_0001/dqc_0001_2017.json [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+        "additional_members": [],
+        "defined_members": [
+            "SubsequentEventMember",
+            "SubsequentEventTypeDomain"
+        ],
+        "excluded_axes": {},
+        "extensions": [],
+        "rule_index": 74
+    }
+}
```
Comment (mindeerickson-wf) [inline dqc_us_rules/resources/DQC_US_0001/dqc_0001_2017.json]: git hub wants a new line.

**Comment by mindeerickson-wf** [comment]
Pass complete.  Looks good to me other than the one comment.

**Comment by joyreistad-wf** [comment]
+1

**Comment by derekgengenbacher-wf** [comment]
+1

**Comment by davidtauriello** [comment]
Dear Workiva Team –



I posted a note to Allen in Skype earlier today but didn’t hear back, so
I’m emailing on the PR thread to loop everyone in –



This commit should be picked into the PR 245 -
https://github.com/davidtauriello/dqc_us_rules/commit/48252f49faaeca406b9538b279bd140292779379
- it contains documentation edits corresponding with the code changes.



Also - I the version on DQC_0001 in the PR seems to have skipped a minor
version increment – I believe it should be 3.3.0 (not 3.4.0), and will
eventually match the release version when the pull is merged to master and
tagged (v3.3.0)



Let me know if you have questions.



Sincerely,

David



David Tauriello
*Vice President, Operations*

(202) 448-1985



*xbrl.us* <http://xbrl.us/> is the national consortium

for the business reporting standard.



*From:* derekgengenbacher-wf [mailto:notifications@github.com]
*Sent:* Wednesday, May 10, 2017 3:03 PM
*To:* DataQualityCommittee/dqc_us_rules
*Cc:* Subscribed
*Subject:* Re: [DataQualityCommittee/dqc_us_rules] Added support for 2017
Taxonomy and fixed issue where incorrect members were being added to
resource file. (#245)



+1

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/DataQualityCommittee/dqc_us_rules/pull/245#issuecomment-300581618>,
or mute the thread

File: dqc_us_rules/resources/DQC_US_0001/dqc_0001_2017.json (hunk 1 above)
Comment (allenbross-wf) [inline dqc_us_rules/resources/DQC_US_0001/dqc_0001_2017.json]: All of the resource files are auto-generated without a newline at the end.

Comment (mindeerickson-wf) [inline dqc_us_rules/resources/DQC_US_0001/dqc_0001_2017.json]: OK.  Thanks

File: dqc_us_rules/_pkg_meta.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 """
 This module contains the revision information for the dqc_us_rules library.
 """
+<<<<<<< HEAD
```
Comment (mindeerickson-wf) [inline dqc_us_rules/_pkg_meta.py]: Should this still be here?

File: dqc_us_rules/_pkg_meta.py [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
 version_info = (3, 2, 1)
+=======
```
Comment (mindeerickson-wf) [inline dqc_us_rules/_pkg_meta.py]: Same?

File: dqc_us_rules/_pkg_meta.py [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+version_info = (3, 3, 0)
+>>>>>>> 48252f49faaeca406b9538b279bd140292779379
```
Comment (mindeerickson-wf) [inline dqc_us_rules/_pkg_meta.py]: Same?

File: dqc_us_rules/_pkg_meta.py [hunk 5]
```diff
@@ -1,9 +1,5 @@
 """
 This module contains the revision information for the dqc_us_rules library.
 """
-<<<<<<< HEAD
```
Comment (mindeerickson-wf) [inline dqc_us_rules/_pkg_meta.py]: Should these still be here?

**Comment by joyreistad-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Duh.  Sorry...

+1

**Comment by jasonleinbach-wf** [comment]
QA + 10 - verified attached doc threw no exceptions.

**Pull request merged**